chrome/content/zotero/xpcom/data/itemFields.js

```diff
@@ -172,7 +173,7 @@ Zotero.ItemFields = new function() {
 		}
 		
 		if (_itemTypeFields[itemTypeID]) {
-			return _itemTypeFields[itemTypeID];
+			return _itemTypeFields[itemTypeID].slice();
```

Comment:
If the concern here is that a consumer could mutate these, maybe calling `Object.freeze()` on the objects when we construct them would be better than copying them every time we return them?